x/twap/keeper_test.go

```diff
@@ -583,6 +583,7 @@ func newThreeAssetOneSidedRecord(time time.Time, accum sdk.Dec, useP0 bool) []ty
 	} else {
 		record.P1ArithmeticTwapAccumulator = accum
 	}
+	record.GeometricTwapAccumulator = accum
```

Comment:
odd that its the same accumulator set for arithmetic and geometric ones, but w/e